tests/e2e/support/objects/runtime/application.ts

```diff
@@ -26,6 +26,12 @@ export class Application {
   }
 
   async getNotificationMessages(): Promise<string[]> {
+    await this.#page.waitForResponse(
+      (resp) =>
+        resp.url().endsWith('notifications?format=json') &&
+        resp.status() === 200 &&
+        resp.request().method() === 'GET'
+    )
```

Comment:
At some cases, this will wait until the next `GET notifications` request is hit.
i.e.
```feature
And "Brian" marks all notifications as read
Then "Brian" should see no notifications
```
Thus, this will increase the feature runtime to some extent.

For `notifications.ocis.feature`
Before
```
0m13.674s (executing steps: 0m13.300s)
Done in 18.33s.
```

After
```
1m13.262s (executing steps: 1m12.941s)
Done in 78.35s.
```

Reply:
Other than this, LGTM

Reply:
I think this is fine. the web sends `GET https://host.docker.internal:9200/ocs/v2.php/apps/notifications/api/v1/notifications?format=json` every 30 seconds automatically, and we don't have any trigger (press a button or anything) to send the request manually. so waiting no more than 30 seconds is not critical..